include/realtime_tools/realtime_helpers.hpp

```diff
@@ -29,6 +29,7 @@
 #ifndef REALTIME_TOOLS__REALTIME_HELPERS_HPP_
 #define REALTIME_TOOLS__REALTIME_HELPERS_HPP_
 
+#include <pthread.h>
```

Comment:
@SENAI-GilmarCorreia Do you need `pthread.h` here?

Reply:
To compile successfully, I really need this.

If it’s not included, I get the following error:

```
C:\path\to\workspace\src\realtime_tools\include\realtime_tools/realtime_helpers.hpp(72,50): error C2065: 'pthread_t': undeclared identifier [C:\path\to\workspace\build\realtime_tools\realtime_tools.vcxproj]
C:\path\to\workspace\src\realtime_tools\include\realtime_tools/realtime_helpers.hpp(72,60): error C2146: syntax error: missing ')' before identifier 'thread' [C:\path\to\workspace\build\realtime_tools\realtime_tools.vcxproj]
  Building Custom Rule C:/path/to/workspace/src/realtime_tools/CMakeLists.txt
  realtime_helpers.cpp
C:\path\to\workspace\src\realtime_tools\include\realtime_tools/realtime_helpers.hpp(72,50): error C2065: 'pthread_t': undeclared identifier [C:\path\to\workspace\build\realtime_tools\thread_priority.vcxproj]
C:\path\to\workspace\src\realtime_tools\include\realtime_tools/realtime_helpers.hpp(72,60): error C2146: syntax error: missing ')' before identifier 'thread' [C:\path\to\workspace\build\realtime_tools\thread_priority.vcxproj]
```

Reply:
So, you had to add it for pthread_t. I've a good solution to this then. I'll try to add it to your PR when I get some time 

So, we can avoid that header for windows